guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -5,8 +5,10 @@ You can use the Leapp tool to upgrade as well as to help detect and resolve issu
 
 .Prerequisites
 ifdef::satellite[]
-* Review Known Issues before you begin an upgrade.
+* Review known issues before you begin an upgrade.
 For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you use an HTTP proxy in your environment, configure the Subscription Manager to use the HTTP proxy for connection.
+For more information, see {RHELDocsBaseURL}9/html-single/upgrading_from_rhel_8_to_rhel_9/index#troubleshooting-resources_troubleshooting[Troubleshooting resources]. 
```

Comment:
I'd move this to the previous bullet point because it is related to the known issues and not to HTTP proxy. Also, I would use a link that points to the Troubleshooting chapter in the RHEL docs as a whole because you want users to find the known issues, however, you're pointing them to the resources, which is a bit confusing.

Reply:
done!